### PR TITLE
Setup test environment

### PR DESCRIPTION
**Pull request by mgu**

### CODE REVIEW

**Comment by ewjoachim** [comment]
@mgu
- tests pass locally & not in the CI
- version seems of (first 10 then 12) but oids seems not to have changed, as if the _on_connect signal had not run.

**Comment by ewjoachim** [comment]
and py3.7 tests keeps getting cancelled and I don't know why.

File: tests/test_integration.py [hunk 1]
```diff
@@ -15,6 +16,10 @@ def test_integration(transactional_db, settings):
...
-    assert str(connection.connection.server_version).startswith("12")
-    assert postgres_signals.get_hstore_oids("default") != hstore
-    assert postgres_signals.get_citext_oids("default") != citext
+    try:
+        assert str(connection.connection.server_version).startswith("12")
+        assert postgres_signals.get_hstore_oids("default") != hstore
+        assert postgres_signals.get_citext_oids("default") != citext
+    finally:
+        settings.DATABASES["default"]["PORT"] = defaultport
+        connection.connect()
```
Comment (ewjoachim) [inline tests/test_integration.py]: not fond of a try/ finally in tests. I'd much rather have a fixture even for a single test.

**Review by mgu — COMMENTED** [review]
LGTM :)

**Comment by mgu** [comment]
cc @PierreDucroquet

**Pull request merged**